test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5190,6 +5190,20 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
                 self.manager(MGR_CMD_DELETE, RSC, id=rescs)
         return True
 
+    def revert_server_attribs(self):
```

Comment:
I don't see the name being changed to revert_server_teardown

Reply:
Sorry Bhroam, I missed to push one file change. Please review it now.